File: Sorting/Python/Bubble_Sort.py
def bubble_sort(arr):
    num=len(arr)
    for a in range (num-1):
        for b in range (num-a-1):
            if(is_small(arr,b,b+1)):
                arr[b],arr[b+1]=arr[b+1],arr[b]

def is_small(arr,a,b):
    return arr[a]>arr[b]


num = int(input("Enter the number of element in the Array: "))
arr = list(map(int,input("Enter the elements of Array: ").split()))

print("UnSorted Array: " , arr)

bubble_sort(arr)

print("Sorted Array: " , arr)

File: Sorting/Bubble_Sort.java
import java.util.Arrays;
import java.util.Scanner;

public class Bubble_Sort {
    public static int[] arr;
    public static int num;
    public static void main(String [] CSECU){
        Scanner Arnab = new Scanner(System.in);
        System.out.print("Enter the Size of the Array: ");
        num = Arnab.nextInt();
        arr= new int[num];
        System.out.print("Enter the Unsorted Array: ");
        for(int a=0;a<num;a++){
            arr[a]= Arnab.nextInt();
        }
        System.out.println("Unsorted Array: "+Arrays.toString(arr));

        arr= Bubble();

        System.out.println("Sorted Array: "+Arrays.toString(arr));
        
    }
    public static int[] Bubble(){
        for(int a=0;a<num-1;a++){
            for(int b=a+1;b<num;b++){
                if(is_Small(a,b)){
                    int temp=arr[a];
                    arr[a]=arr[b];
                    arr[b]=temp;
                }
            }
        }
        return arr;
    }
    public static boolean is_Small(int a,int b){
        return arr[a]>arr[b];
    }
}
